src/google/protobuf/message.h

```diff
@@ -740,7 +740,7 @@ class PROTOBUF_EXPORT Reflection final {
   // long as the message is not destroyed.
   //
   // Note that to use this method users need to include the header file
-  // "net/proto2/public/reflection.h" (which defines the RepeatedFieldRef
```

Comment:
Could you perhaps change it to just reflection.h without the full path? For historical reasons we have a somewhat different path structure within Google, which is why there are some comments like this referring to net/proto2/public.